src/operationDefinitions/types.ts

```diff
@@ -5,20 +5,36 @@
 
 import { Router } from 'express';
 import { SystemOperation, TypeOperation } from 'fhir-works-on-aws-interface';
+import ResourceHandler from '../router/handlers/resourceHandler';
 
 export interface OperationDefinitionImplementation {
     /**
      * url of the corresponding OperationDefinition resource
      */
-    canonicalUrl: string;
+    readonly canonicalUrl: string;
+
+    /**
+     * url path used to invoke the operation
+     */
+    readonly path: string;
+
+    /**
+     * methods supported by this operation e.g GET, POST
+     */
+    readonly httpMethods: string[];
```

Comment:
In the Interface package we used the term 'verb' instead of 'method': https://github.com/awslabs/fhir-works-on-aws-interface/blob/mainline/src/authorization.ts#L23